opts/opts_test.go

```diff
@@ -445,7 +446,16 @@ func TestValidateHost(t *testing.T) {
 		"tcp://invalid":      "Invalid bind address format: invalid",
 		"tcp://invalid:port": "Invalid bind address format: invalid:port",
 	}
+	var (
+		defaultHTTPHost = "tcp://127.0.0.1:2375"
+		defaultHOST     = "unix:///var/run/docker.sock"
+	)
+	if runtime.GOOS == "windows" {
+		defaultHOST = defaultHTTPHost
+	}
```

Comment:
I was under the impression that both windows client and server would use tcp. (and I've been contemplating OSX too :) )